### PR TITLE
Use File instead of raw InputStream

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/storage/connector/HttpFileUploaderIntegrationTest.java [hunk 1]
```diff
@@ -84,14 +83,15 @@ static void beforeAll() throws IOException, InterruptedException
...
     @Test
-    void testUpload() throws IOException
+    void testUpload()
     {
         // given
         List<String> initialBucketContent = s3StorageMock.listBucketContent(BUCKET_NAME);
-        @Cleanup
-        InputStream fileInputStream = new ByteArrayInputStream(OBJECT_CONTENT.getBytes());
+
+        File fileToUpload = new File(new ByteArrayInputStream(OBJECT_CONTENT.getBytes()));
```
Comment (krdabrowski) [inline live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/storage/connector/HttpFileUploaderIntegrationTest.java:91]: You are still constructing here new `InputStream` so maybe you should leave the `@Cleanup` annotation? 🤔

Comment (Pawel-608) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/storage/connector/HttpFileUploaderIntegrationTest.java]: fixed, but idk if we need it

Comment (Pawel-608) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/storage/connector/HttpFileUploaderIntegrationTest.java]: guess we need it :)

**Review by krdabrowski — APPROVED** [review]
LGTM

**Pull request merged**